Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN conda update conda --yes \
     branca=0.3.1 \
```

Comment:
We are pinning branca because of an issue with folium. but geopandas was not installing correctly prior to pinning branca